automatic/sysinternals/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,6 @@
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+
+if (Test-Path $toolsPath\chocoUninstall.ps1) {
+    . $toolsPath\chocoUninstall.ps1
```

Comment:
Instead of sourcing a powershell script here, you should rather iterate over a list of paths stored in a text file (as mentioned above).